cms/djangoapps/contentstore/courseware_index.py

```diff
@@ -621,7 +621,7 @@ def index_about_information(cls, modulestore, course):
             if section_content:
                 if about_information.index_flags & AboutInfo.ANALYSE:
                     analyse_content = section_content
-                    if isinstance(section_content, basestring):
+                    if isinstance(section_content, str):
```

Comment:
@feanil This change might break the code if any unicode string appear.
```
isinstance(section_content, string_types[0]) or isinstance(section_content, string_types)

Reply:
@feanil 
```
Python 2.7.10 (default, Oct  6 2017, 22:29:07)
[GCC 4.2.1 Compatible Apple LLVM 9.0.0 (clang-900.0.31)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> s ='a str'
>>> su = u'a u-str'
>>> isinstance(s, basestring)
True
>>> isinstance(su, basestring)
True
>>> isinstance(s, str)
True
>>> isinstance(su, str)
False
>>>

Reply:
@awais786 perhaps a safer thing is https://six.readthedocs.io/#six.string_types ?

Reply:
@feanil let me try this.